templates/bin_setup.erb

```diff
@@ -7,8 +7,8 @@
 set -e
 
 # Set up Ruby dependencies via Bundler
-gem list bundler --installed > /dev/null || gem install bundler
-bundle install
+command -v bundle > /dev/null || gem install bundler
```

Comment:
We use `gem install --conservative bundler` in Paperback's `bin/setup`
script. Maybe @trevororeilly has thoughts on it.

On Thu, Jan 22, 2015 at 2:10 PM, Joe Ferris <notifications@github.com>
wrote:

> In templates/bin_setup.erb
> <https://github.com/thoughtbot/suspenders/pull/497#discussion_r23414815>:
>
> > @@ -7,8 +7,8 @@
> >  set -e
> >
> >  # Set up Ruby dependencies via Bundler
> > -gem list bundler --installed > /dev/null || gem install bundler
> > -bundle install
> > +command -v bundle > /dev/null || gem install bundler
>
> I think it is related.
>
> The gem list bundler line being changed here is to make sure Bundler gets
> installed. It looks like RubyGems has that functionality built-in (install
> a gem, only if it's missing), and the flag for it is --conservative.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/497/files#r23414815>.
>